src/ducks/Alert/components/AlertModalContent/StepsContent/NextStep/NextStep.module.scss

```diff
@@ -0,0 +1,12 @@
+.wrapper {
+  height: 24px;
+  padding: 0;
+
+  &:hover {
+    color: var(--jungle-green-light-3);
+  }
+
+  .icon {
+    margin-left: 8px;
+  }
```

Comment:
Better to not nest the classes, since it can be assigned directly using `css modules`